tests/general/KeywordBearTest.py

```diff
@@ -14,3 +23,99 @@
     invalid_files=("test line todo",
                    "test line warNING"),
     settings={"keywords": "todo, warning"})
+
+
+class KeywordBearTodoTest(unittest.TestCase):
```

Comment:
hm I would call this one `KeywordBearDiffTest` or `KeywordBearPatchTest`, as "Todo" is just a keyword itself^^